app/components/view-goal/view-goal.jsx

```diff
@@ -1,8 +1,48 @@
 import React from 'react';
+import GoalTileComponent from '../goal-tile.jsx'
+import ProgressBarComponent from './progress-bar.jsx'
+
+const viewGoal = ( { currentGoal } ) => {
+  const rateGoal = (goal) => {
+    console.log("hello");
```

Comment:
Hmm.. It is better to get actions / reducers in place, then to build the view, then to be using fake actions..